runtime/lua/vim/shared.lua

```diff
@@ -409,6 +398,18 @@ end
 ---@param ... table Two or more tables
 ---@return table : Merged table
 function vim.tbl_extend(behavior, ...)
+  if behavior ~= 'error' and behavior ~= 'keep' and behavior ~= 'force' then
```

Comment:
Duplicate code -- refactor into a local validate function?

Reply:
Done. And hoisted out another argument check.